src/sentry/integrations/slack/utils.py

```diff
@@ -409,9 +420,9 @@ def get_channel_id(organization, integration_id, name):
 def get_channel_id_with_timeout(integration, name, timeout):
     """
     Fetches the internal slack id of a channel.
-    :param organization: The organization that is using this integration
-    :param integration_id: The integration id of this slack integration
+    :param integration: The slack integration
     :param name: The name of the channel
+    :param timeout: Our self-imposed time limit (10s).
```

Comment:
@mgaeta Nit: we use a different value of `timeout` for the async case (3 minutes)